distributor-widget/reference.md

```diff
@@ -148,7 +148,7 @@ As the third parameter, `Mews.Distributor` accepts optional configuration.
 Mews.Distributor(
     { configurationIds: ['aaaa-bbbb-cccc-dddd-eeeeeeee'] }, 
     function (dst) { dst.open(); },
-    { dataBaseUrl: 'https://demo.mews.li' }
+    { dataBaseUrl: 'https://api.mews-demo.com' }
```

Comment:
Not sure if typo or on purpose?
```suggestion
    { dataBaseUrl: 'https://app.mews-demo.com' }
```

Reply:
`api` is intentional here that is URL where data are provided. In theory `app.mews-demo.com/api` will not exist